modules/ROOT/pages/installation/sso.adoc

```diff
@@ -0,0 +1,271 @@
+= Single Sign-On (SSO) Configuration
+:description: This section describes the configuration steps to enable Single Sign-On (SSO) for NOM server
+
+== Overview
+
+NOM supports SSO for authentication using external Identity Providers (IdP) that support OAuth2/OIDC implicit authentication flow.
+Currently, Microsoft Entra ID (formerly Azure Active Directory), Google Identity and Okta are supported.
+More IdP types and alternative authentication flows may be added in future NOM releases.
+SSO can be used as the only authentication method or in combination with username/password login.
+When both methods are enabled, users can choose the login method on the NOM login page.
+
+== Configuration
+
+One or more SSO configs can be set up containing the options needed to perform SSO authentication with an external Identity Provider (IdP).
+
+Configurations are identified by a name, which is used to reference the configuration in the environment variable names or command line arguments.
+The name can contain only letters and digits. In the reference below, the placeholder `<CONFIG>` is used to indicate where the configuration name should be used.
+
+The configuration contains options needed to obtain an access token on the NOM frontend side, validate the access token on the NOM backend side and map token claims to an SSO user in NOM, including assign a NOM role to the SSO user.
+
+Only the options needed for obtaining the token are send to the frontend.
+
+Configurations are validated during the startup of NOM server.
+If a configuration is invalid, the errors are reported in the logs.
+
+=== Common options
+
+[cols="<,<,<, <",options="header"]
+|===
+| Command line argument
+| Environment variable name
+| Description
+| Example value
+
+| `app.sso.active-configurations`
+| `APP_SSO_ACTIVE_CONFIGURATIONS`
+| Lists comma-separated names of configurations that should be offered as login options on the NOM login page.
+| `myConfig1,myconfig2`
+
+| `app.username-password-login.enabled`
+| `APP_USERNAME_PASSWORD_LOGIN_ENABLED`
+| Username/password login can be enabled or disabled, enabling SSO login either be offered additionally or to be the only login method.
+If disabled, the pages "NOM Administrators" and "Profile" will not be available in the NOM UI.
+| `false`
+
+| `app.sso.configurations.<CONFIG>.idp-type`
+| `APP_SSO_CONFIGURATIONS_<CONFIG>_IDP_TYPE`
```

Comment:
We could considering breaking the table here and prepending the second one with a disclaimer similar to:

> In the reference below, the placeholder `<CONFIG>` is used to indicate where the configuration name should be used.

Maybe also mention that `<CONFIG>` in the example would be `myConfig1` and `myconfig2`.

We've seen users make silly mistakes around this type of substitution structure before..